### ISSUE TITLE
Its possible to create empty fleets and move around with them

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/open-ig#12]
```
What steps will reproduce the problem?
1. create a fleet and dont assign any ships to it
2.
3.

What is the expected output? What do you see instead?



What version of the product are you using? On what operating system?

0.90



```

Original issue reported on code.google.com by `t.eckfel...@gmx.de` on 16 Apr 2011 at 3:05

**Comment by GoogleCodeExporter** [comment]
```
Bug. Will be fixed in v0.91
```

Original comment by `akarn...@gmail.com` on 16 Apr 2011 at 5:41
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Delivered in v0.91
```

Original comment by `akarn...@gmail.com` on 14 Jun 2011 at 4:06
- Changed state: **Fixed**
